src/events/cd-event-add-child-ticket.vue

```diff
@@ -0,0 +1,182 @@
+
+      <label>{{ $t('Gender') }}</label>
+      <div class="cd-child-ticket__gender-selector">
+        <select class="cd-child-ticket__select-box form-control" data-vv-name="gender" data-vv-validate-on="blur" v-validate="'required'" v-model="genderSelect">
+          <option value="" selected data-default disabled>{{ $t('Select an option') }}</option>
+          <option value="male">{{ $t('Male') }}</option>
+          <option value="female">{{ $t('Female') }}</option>
+          <option value="prefer not to answer">{{ $t('Prefer not to answer') }}</option>
+          <option value="specify">{{ $t('Specify Identity') }}</option>
+        </select>
+        <input v-show="specifyGender" class="form-control" v-model="genderInput" :placeholder="$t('Identify as...')" data-vv-name="gender" data-vv-validate-on="blur" v-validate="specifyGender ? 'required' : ''"/>
+      </div>
+      <p class="cd-child-ticket__gender-err text-danger" v-show="errors.has('gender:required')">{{ $t('Gender is required') }} <a v-on:click="showWhy">{{ $t('Why is this required? Click Here To Find Out More') }}</a><p v-show="whyGender && errors.has('gender:required')">{{ $t(`We want to provide activities that appeal to people regardless of their gender. To check how well we are succeeding, we'd like to find out whether or not people of different genders are equally likely to take part.`) }}</p></p> 
+
+      <label>{{ $t('Ticket') }}</label>
+      <div class="cd-child-ticket__ticket-selector"> 
+        <multiselect v-model="tickets" :options="childTickets" group-label="name" group-values="tickets" :multiple="true" :searchable="false" :group-select="false" :placeholder="$t('Select Event Tickets')" track-by="id" label="name" @close="onTicketTouch" ></multiselect>
+      </div>
+      <p class="cd-child-ticket__ticket-select-err text-danger" v-show="invalidTicket">{{ $t('Ticket selection is required') }}</p>  
+    </div>
+  </div>
+</template>
+
+<script>
+  import VueDobPicker from 'vue-dob-picker';
+  import Multiselect from 'vue-multiselect';
+
+  export default {
+    name: 'ChildTicket',
+    props: ['sessions', 'eventId', 'event'],
+    components: {
+      VueDobPicker,
+      Multiselect,
+    },
+    data() {
+      return {
+        firstName: '',
+        surname: '',
+        dob: null,
+        genderSelect: '',
+        genderInput: '',
+        whyGender: false,
+        ticketTouch: false,
+        tickets: '',
+      };
+    },
+    methods: {
+      showWhy() {
+        this.whyGender = true;
+      },
+      onTicketTouch() {
+        this.ticketTouch = true;
+      },
+    },
+    filters: {
+      addPosession(firstName) {
+        if (firstName) {
+          return ` - ${firstName}`;
+        }
+        return firstName;
+      },
+    },
+    computed: {
+      childTickets() {
+        return this.sessions.map(session => ({
+          description: session.description,
+          entity$: session.entity$,
```

Comment:
You can drop the entity$ from there. It shouldn't have been there in the first place and won't be.

Reply:
Okeedokes!